server/src/variants.go

```diff
@@ -249,6 +256,13 @@ func variantIsCardTouched(variantName string, clue Clue, card *Card) bool {
 			return false
 		}
 
+		if variant.OddsAndEvens {
+			if clue.Value == 1 {
```

Comment:
is there a constant for this

Reply:
For `clue.Value == 1`? No

Reply:
yes there is https://github.com/Hanabi-Live/hanabi-live/blob/main/server/src/constants.go#L32-L35

Reply:
`clue.Value == 1` means that the player clued `1` to someone else. This is checked because in Odds And Evens a rank clue can be either `1` for odd or `2` for even ranked cards.

Reply:
> yes there is https://github.com/Hanabi-Live/hanabi-live/blob/main/server/src/constants.go#L32-L35

This defines the clue.Type

Reply:
oh i see